build.install4j

```diff
@@ -45,7 +45,7 @@
     <components />
   </files>
   <launchers>
-    <launcher name="TripleA" id="33" customizedId="" external="false" excludeFromMenu="false" unixMode="755" menuName="TripleA-${compiler:sys.version}" icnsFile="" customMacBundleIdentifier="false" macBundleIdentifier="" swtApp="false" fileset="" macBundleBinary="JavaApplicationStub" addMacEntitlements="false" macEntitlementsFile="" useCustomMacosExecutableName="false" customMacosExecutableName="" useJavaMinVersionOverride="false" javaMinVersionOverride="" useJavaMaxVersionOverride="false" javaMaxVersionOverride="">
+    <launcher name="TripleA" id="33" customizedId="" external="false" excludeFromMenu="false" unixMode="755" unixAutoStart="true" menuName="TripleA-${compiler:sys.version}" icnsFile="" customMacBundleIdentifier="false" macBundleIdentifier="" swtApp="false" fileset="" macBundleBinary="JavaApplicationStub" addMacEntitlements="false" macEntitlementsFile="" useCustomMacosExecutableName="false" customMacosExecutableName="" useJavaMinVersionOverride="false" javaMinVersionOverride="" useJavaMaxVersionOverride="false" javaMaxVersionOverride="">
```

Comment:
Found it.  It has to do with auto-starting installed services (i.e. registering them with `systemd`).  It appears in 6.1.5:

![install4j-6 1 5-launcher-unix-options](https://cloud.githubusercontent.com/assets/4826349/26690900/1c70822a-46c9-11e7-87f1-62f5864dcdff.png)

But is not present in 6.1.2:

![install4j-6 1 2-launcher-unix-options](https://cloud.githubusercontent.com/assets/4826349/26690904/2373fda4-46c9-11e7-8864-16789433fc8a.png)

So, it sounds like it's something we don't need to worry about.